common/static/common/templates/discussion/thread-list-item.underscore

```diff
@@ -21,6 +21,9 @@
       <span class="icon fa <%= icon_class %>" aria-hidden="true"></span>
     </div><div class="forum-nav-thread-wrapper-1">
       <span class="forum-nav-thread-title"><%- title %></span>
+      <% if (!noThreadBodyPreview){ %>
```

Comment:
Nit: I'm thinking this parameter would be simpler if it were not a double negative. Maybe call it ``showThreadPreview``.